sources/Debate-frontend/src/app/services/auth.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { UserLogIn } from '../dto/user-log-in';
+import { UserDto } from '../dto/user.dto';
+import { Observable } from 'rxjs/internal/Observable';
+import { HttpClient } from '@angular/common/http';
+import { CookieService } from 'ngx-cookie-service';
+import { map } from 'rxjs/operators';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  private endpoint = 'http://localhost:8080/api';
+  userLogin: UserLogIn;
+  userDto: UserDto;
+  private isLoggedIn = new BehaviorSubject<boolean>(false);
+
+  constructor(private http: HttpClient, private cookieService: CookieService) {}
+
+  logToAccount(user: UserDto): Observable<UserLogIn> {
+    return this.http
+      .post<UserLogIn>(`${this.endpoint}/userLogin/logIn`, user)
+      .pipe(
+        map((usrLogin) => {
```

Comment:
```suggestion
        tap((usrLogin) => {
```

Should be tap instead of map as no mapping actually takes place